client/src/components/dataList/SpeciesInfo.js

```diff
@@ -0,0 +1,293 @@
+import { useEffect, useState } from "react";
+import React from "react";
+import { useReducer } from "react";
+// import DeleteSpecies from "./DeleteSpecies";
+import deleteIcon from "./deleteIcon.png"
+// create useReducer for changing values
+import EditSpecies from "./edit/EditSpecies";
+
+const reducer = (state, action) => {
+    console.log(action, 'this is the action');
+    switch (action.type) {
+        case 'editID':
+            console.log('Logged if the editName action is being dispatched');
+            //it updates the name to the input value
+            return { ...state, id: action.payload };
+
+        case 'editCommon_name':
+            return { ...state, common_name: action.payload };
+
+        case 'editScientificName':
+            return { ...state, scientific_name: action.payload };
+
+        case 'editPopulation':
+            return { ...state, population: action.payload };
+
+        case 'editConservationStatus':
+            return { ...state, conservation_status: action.payload };
+        case 'editCreatedOn':
+            return { ...state, created_on: action.payload };
+        case 'clearForm':
+            return { id: "", common_name: '', scientific_name: '', population: '', conservation_status: '', created_on: '' };
+        default:
+            return state;
+    }
+};
+const SpeciesInfo = () => {
+    const [species, setSpecies] = useState([]);
+
+    //get species data table
+    const getSpecies = async () => {
+        const response = await fetch(
+            `http://localhost:4040/species`
+        );
+        const specie = await response.json();
```

Comment:
Funny enough `species` is the signular and plurar form. Using non-words as var names make the reader wonder if it is a typo. So I would recommend just using `species` throughout.

Reply:
In this case to make it clear there is more than one I would probably call this `speciesList`.